intTests/test0019_jss_switch_statement/test.sh

```diff
@@ -1,7 +1,3 @@
 set -e
 
-$JSS Test1
```

Comment:
Now that all of the `jss`-related bits are gone, it doesn't make much sense to refer to this subdirectory with the name `test0019_jss_switch_statement`. Perhaps we should rename it.

Reply:
Yeah, I was wondering about that too. I completely removed the other `jss` tests, but this one seemed to have a non-`jss` part with some non-trivial content, so I thought it should remain. We should definitely change the name; perhaps `test0019_jvm_switch_statement` would do nicely.

There are also a few mentions of JSS in comments and READMEs of some of the other integration tests; I should probably edit those as well.